pandas/core/internals/managers.py

```diff
@@ -1780,23 +1780,23 @@ def form_blocks(arrays, names, axes):
 
     if len(items_dict["CategoricalBlock"]) > 0:
         cat_blocks = [
-            make_block(array, klass=CategoricalBlock, placement=[i])
+            make_block(array, klass=CategoricalBlock, placement=i)
             for i, _, array in items_dict["CategoricalBlock"]
         ]
         blocks.extend(cat_blocks)
 
     if len(items_dict["ExtensionBlock"]):
 
         external_blocks = [
-            make_block(array, klass=ExtensionBlock, placement=[i])
+            make_block(array, klass=ExtensionBlock, placement=i)
```

Comment:
could pass `slice(i, i+1)` and avoid editing the cython

Reply:
See Jeff's comment above, he asked the opposite (I originally passed the slice here in the first version of this PR)

Reply:
hah, makes sense